server/src/main/java/org/apache/druid/catalog/model/CatalogUtils.java

```diff
@@ -68,12 +68,20 @@ public static Granularity asDruidGranularity(String value)
     if (Strings.isNullOrEmpty(value) || value.equalsIgnoreCase(DatasourceDefn.ALL_GRANULARITY)) {
```

Comment:
Are there tests for this static utility or perhaps something more end-to-end? If so, it'd be nice if we could add some tests with granularity strings like `HOUR` and `DAY` for the enhancement

Reply:
updated existing integration tests to test all formats `P1D`, `DAY`, `ALL`